src/homework04.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+# coding: utf8
+
+"""
+EPAM python q4 homework 04
+Sergey Streltsov 2019-11-12
+"""
+
+
+import math
+import time
+
+
+def save_history(f_name):
+    """History decorator
+
+    Create a decorator that stores the results of function calls and the
+    time it takes to get the result during program startup
+    """
+    history = list()
+
+    def wrapper(*args, **kwargs):
+        start_time = int(round(time.time() * 1000))
```

Comment:
why can't we just use time.time and if we need some conversion than make it in the end?

Reply:
ok, I agree, move conversion to the end